tools/fixFormat.js

```diff
@@ -180,6 +174,24 @@ function fixFormat() {
     }
   }
 
+  function adaptLicenseName(pkgName, licenseName, isRecognized) {
+    if (isRecognized == 1) {
+      if (licenses.indexOf(licenseName) !== -1) {
+        return licenseName;
+      } else if (adapt(licenseName)) {
+        return adapt(licenseName);
+      } else {
+        unRecognizedLicense(pkgName, licenseName);
```

Comment:
Let's make this return `licenseName` in addition to logging the error. We don't want to assign `undefined` to `pkg.license` in `fixLicense` function.